internal/gencli/testdata/manage-todos.want

```diff
@@ -0,0 +1,104 @@
+// Code generated. DO NOT EDIT.
+
+package main
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/golang/protobuf/jsonpb"
+	"github.com/spf13/cobra"
+
+	todopb "github.com/googleapis/todo/generated"
+)
+
+var ManageTodosInput todopb.Todo
+
+var ManageTodosFromFile string
+
+var ManageTodosOutFile string
+
+func init() {
+	TodoServiceCmd.AddCommand(ManageTodosCmd)
+
+	ManageTodosCmd.Flags().StringVar(&ManageTodosFromFile, "from_file", "", "Absolute path to JSON file containing request payload")
+
+	ManageTodosCmd.Flags().StringVar(&ManageTodosOutFile, "out_file", "", "Absolute path to a file to pipe output to")
+	ManageTodosCmd.MarkFlagRequired("out_file")
+
+}
+
+var ManageTodosCmd = &cobra.Command{
+	Use:   "manage-todos",
+	Short: "manage todos live",
+	Long:  "manage todos live by creating and updating todos as they change",
+	PreRun: func(cmd *cobra.Command, args []string) {
+
+	},
+	RunE: func(cmd *cobra.Command, args []string) (err error) {
+
+		in := os.Stdin
+		if ManageTodosFromFile != "" {
+			in, err = os.Open(ManageTodosFromFile)
+			if err != nil {
+				return err
+			}
+
+		}
+
+		stream, err := TodoClient.ManageTodos(ctx)
+
+		out, err := os.OpenFile(ManageTodosOutFile, os.O_APPEND|os.O_WRONLY, os.ModeAppend)
+		if err != nil {
+			return err
+		}
+
+		// start background stream receive
+		go func() {
+			var res *todopb.Todo
+			for {
+				res, err = stream.Recv()
+				if err != nil {
+					return
+				}
+
+				str := res.String()
+				if OutputJSON {
+					d, _ := json.MarshalIndent(res, "", "  ")
+					str = string(d)
+				}
+				fmt.Fprintln(out, str)
+			}
+		}()
+
+		if Verbose {
+			fmt.Println("Client stream open. Close with blank line.")
+		}
+
+		scanner := bufio.NewScanner(in)
+		for scanner.Scan() {
+			input := scanner.Text()
+			if input == "" {
+				break
```

Comment:
Shouldn't this be `continue`? Presumably you sometimes have blank lines?

Reply:
I made the decision to say that a blank line ends the client stream, requiring an entire message to be provided on within a single line break. So this was on purpose. Usage similar to the example I gave for file input on an earlier comment.

Wdyt? Should it look for a close sequence instead?

Reply:
I opine that blank line ending the stream sounds a little funny. It seems like you should be able to do something like

gcli <<EOF
foo1
foo2
foo3

foo4
foo5
foo6
EOF
```
and use the line break as a visual hint for some kind of logical separation. I admit I don't have a concrete use case in mind. If you think blank line is the right approach I won't fight it very hard.

Reply:
I understand where you are coming from. I will brain storm and address in a separate PR!